include/revng/Model/StatementComment.h

```diff
@@ -12,6 +12,9 @@ name: StatementComment
 type: struct
 
 fields:
+  - name: Index
+    type: uint64_t
+
```

Comment:
Doc: "The index of the comment".

Reply:
Done, both here and in the other PR sharing these commits.